rust/tool/systemd/Cargo.toml

```diff
@@ -12,6 +12,9 @@ log = { version = "0.4.18", features = ["std"] }
 clap = { version = "4.3.1", features = ["derive"] }
 lanzaboote_tool = { path = "../shared" }
 indoc = "2.0.1"
+nix-base32 = "0.1.1"
```

Comment:
The advantage of the Nix encoding is that we can assume that every user has a tool that can handle hashes in this format already available (namely, Nix). And `nix-base32` is really tiny, it looks more "done" than "abandoned" to me.

Reply:
There is a 3 year old pre-release version that never got turned into an actual release. That's abandoned in my books.

Reply:
> There is a 3 year old pre-release version that never got turned into an actual release

… and that actually doesn't contain any code.

Reply:
This does not convince me that the crate is "done". On another note, I've yet to see any code that was just "done". Anyways, even if it is in fact done, I strongly prefer an actually standardised encoding over the weird nix encoding. Please change it.

Reply:
https://github.com/kolloch/nix-base32/blob/42f5544e51187f0c7535d453fcffb4b524c99eb2/src/lib.rs#L11 This code is so simple that I'm not sure how it would not be "done". I will change it anyway, with all the drawbacks that brings.